views/pod.html

```diff
@@ -15,7 +15,7 @@
     <dt>Phase</dt>
     <dd>{{resource.status.phase}}</dd>
     <dt>Node</dt>
-    <dd>{{resource.status.host}}</dd>
+    <dd>{{resource.spec.host}}</dd>
```

Comment:
I dont know why i did this differently from our browse page, ours looks like:

`{{pod.spec.host || 'unknown'}} <span ng-if="pod.status.hostIP && pod.spec.host != pod.status.hostIP">({{pod.status.hostIP}})`